src/Signer/Ecdsa/EccAdapter.php

```diff
@@ -1,118 +0,0 @@
-<?php
-declare(strict_types=1);
-
-namespace Lcobucci\JWT\Signer\Ecdsa;
-
-use GMP;
-use InvalidArgumentException;
-use Mdanter\Ecc\Crypto\Key\PrivateKeyInterface;
-use Mdanter\Ecc\Crypto\Key\PublicKeyInterface;
-use Mdanter\Ecc\Crypto\Signature\Signer;
-use Mdanter\Ecc\Curves\NistCurve;
-use Mdanter\Ecc\EccFactory;
-use Mdanter\Ecc\Math\GmpMathInterface;
-use Mdanter\Ecc\Primitives\GeneratorPoint;
-use Mdanter\Ecc\Random\RandomGeneratorFactory;
-use Mdanter\Ecc\Random\RandomNumberGeneratorInterface;
-use function array_key_exists;
-
-class EccAdapter
```

Comment:
this BC break requires documentation

Reply:
This class was always considered as internal, but we were not using `@internal` yet. I'm not sure if this really need to be stated as BC-break :(